Sources/System/FileHelpers.swift

```diff
@@ -122,4 +122,18 @@ extension FileDescriptor {
       return .success(buffer.count)
     }
   }
+
+  /// Return the current size of the file.
+  ///
+  /// - Returns: The current size of the file, in bytes.
+  @_alwaysEmitIntoClient
+  @discardableResult
```

Comment:
Looking at this again, this probably shouldn't be discardable.

Reply:
Addressed as part of:

* d90bf85 `fixup: Move FileDescriptor.fileSize() to an internal extension in SystemTests`